data_preparation/crop_statistics_ML/README.md

```diff
@@ -1,47 +1,47 @@
-# Compagnie Malienne pour le Developpement des Textiles (CMDT) Monitoring Operational Maize Yield Statistics
-
-## Short description
-The data derived comes from CMDT Monitoring Operational - a Malian Company for the Development of Textiles located in Southern Mali. It consists of 5 CMDT regions and 28 CMDT sectors. It provides key observations from the maize yield data for the different zones/regions and sectors from 1990 to 2017. There is significant spatial and temporal variation in maize productivity in the region, likely driven by differences in agro-climatic conditions, management practices, and potential data limitations. 
-
-## Link
-
-
-
-## Publisher
-CMDT operational monitoring
-
-## Dataset owner
-ICRISAT
-
-## Data card author
-Janet Mutuku, Celeste Tchampi, PCS Traore
-
-## Dataset overview
-**Crops**: Maize
-
-**Variables** [unit]: Year, cmdt_zone, cmdt_sector, yield(mt), adm2_pcode, production(mt), area, other derived stats
-
-
-**Temporal coverage**: 1990-2017 (for yield), 1990-1999 (production(mt) and area (ha))
-
-
-**Temporal resolution**: Yearly
-
-**Spatial resolution**: Arrondissement-level (sectors) (level 3 administrative boundaries)
-
-**Date Published**: N/A
-
-**Data Modality**: Tabular data
-
-## Provenance
-**Version**: N/A <br>
-
-
-## License
-The publisher permits the download and usage of the data for non-commercial purposes.
-
-## Additional information
-
-
-## References
+# Compagnie Malienne pour le Developpement des Textiles (CMDT) Monitoring Operational Maize Yield Statistics
+
+## Short description
+The data derived comes from CMDT Monitoring Operational - a Malian Company for the Development of Textiles located in Southern Mali. It consists of 5 CMDT regions and 28 CMDT sectors. It provides key observations from the maize yield data for the different zones/regions and sectors from 1990 to 2017. There is significant spatial and temporal variation in maize productivity in the region, likely driven by differences in agro-climatic conditions, management practices, and potential data limitations. 
+
+## Link
+
+
+
+## Publisher
+CMDT operational monitoring
+
+## Dataset owner
+ICRISAT
+
+## Data card author
+Janet Mutuku, Celeste Tchampi, Pierre C. Sibiry Traore
+
+## Dataset overview
+**Crops**: Maize
+
+**Variables** [unit]:'country_code', 'cmdt_zone', 'cmdt_sector', 'adm_id', 'season_name',
+       'harvest_year', 'crop_name', 'production', 'planted_area', 'yield'

```

Comment:
These are variables in the original data, not in the processed data. This should go back to the previous one.

Reply:
These variables have been excluded. The readme in the shapefile contains these: 'country_code', 'cmdt_zone', 'cmdt_sector', 'adm_id'